Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/Header.cshtml

```diff
@@ -227,20 +227,32 @@
                     }
                 }
             }
-            else
-            {
-                // TODO: Update this once the content is agreed in #232329.
-                newlineRequired = true;
-                <span>GIAS no longer displays Ofsted inspection ratings</span>
-            }
 
             @if (newlineRequired)
             {
                 <br/>
             }
-            <a href="@Model.OfstedReportUrl" target="_blank" rel="noreferrer noopener">
-                Ofsted report (opens in new tab)
-            </a>
+
+            <div class="govuk-!-margin-bottom-3">
+                <a href="@Model.OfstedReportUrl" target="_blank" rel="noreferrer noopener">
+                    Ofsted report (opens in new tab)
+                </a>
+            </div>
+
+            @if (!Model.ShowOfstedRatings)
+            {
+                <div>GIAS no longer displays Ofsted inspection ratings</div>
```

Comment:
Depending on feedback, it might be that this line is not wanted and will need to be removed.

Reply:
Feedback just received to remove this line -- additional commit is incoming now.